contracts/bridge/Bridge.sol

```diff
@@ -83,74 +85,169 @@ contract Bridge is IERC20BridgeReceiver, IERC721BridgeReceiver, Ownable, BridgeF
         uint64 handleNonce
     );
 
-    // start allows the value transfer request.
-    function start() external onlyOwner {
-        isRunning = true;
+    modifier onlySigners()
+    {
+        require(msg.sender == owner() || signers[msg.sender]);
+        _;
     }
 
-    // stop prevent the value transfer request.
-    function stop() external onlyOwner {
-        isRunning = false;
+    // start allows the value transfer request.
+    function start(bool _status, uint64 _requestNonce)
+        external
+        onlySigners
+    {
+        onlySequentialNonce(TransactionType.Governance, _requestNonce);
+        bytes32 voteKey = keccak256(abi.encodePacked(this.start.selector, _status, _requestNonce));
```

Comment:
`abi.encodePacked` is necessary?
Please consider this. I think we need to compare the hashing ways.
https://ethereum.stackexchange.com/questions/48827/does-keccak256-and-sha3-reproduce-the-same-hash

Reply:
It causes compiler warning.

Reply:
https://ethereum.stackexchange.com/questions/50592/what-does-warning-this-function-only-accepts-a-single-bytes-argument-please